### PR TITLE
S 8 to master

### PR DESCRIPTION
**Pull request by shermet5**

### CODE REVIEW

**Review by ocheretenyukVadym — APPROVED** [review]

**Review by ocheretenyukVadym — APPROVED** [review]

File: src/Components/TicketPage/NewTicketPage/NewTicketPage.jsx [hunk 1]
```diff
@@ -1,11 +1,29 @@
...
-const NewTicketPage = (props) => {
-    return(
-        <div>
 
-        </div>
-    )
+const NewTicketPage = (props) => {
+  const [input, setInput] = useState('');
+  const handleChange = e => {
+    setInput(e.target.value);
```
Comment (vansisto) [inline src/Components/TicketPage/NewTicketPage/NewTicketPage.jsx:7]: setInput('');

File: src/Components/TicketPage/NewTicketPage/NewTicketPage.jsx [hunk 2]
```diff
@@ -1,11 +1,29 @@
...
+  }
+  const handleSubmit = e => {
+    e.preventDefault();
+      props.createNewTicket(input)
+
```
Comment (vansisto) [inline src/Components/TicketPage/NewTicketPage/NewTicketPage.jsx:12]: -

File: src/Components/TicketPage/NewTicketPage/NewTicketPage.jsx [hunk 3]
```diff
@@ -1,11 +1,29 @@
...
+    e.preventDefault();
+      props.createNewTicket(input)
+  
+  }
+  return (
+    <div className='container'>
+        <input
+          value={input}
+          type='text'
+          name='text'
```
Comment (vansisto) [inline src/Components/TicketPage/NewTicketPage/NewTicketPage.jsx:19]: -----------

**Review by vansisto — CHANGES_REQUESTED** [review]

**Comment by vansisto** [comment]
ytr

**Pull request closed without merging**